### ISSUE TITLE
SSI: failed to move target

### ISSUE THREAD

**Issue by jgrund** [whamcloud/integrated-manager-for-lustre#934]
Seen in: https://build.whamcloud.com/job/integration-tests-shared-storage-configuration/1762/

In test:

tests.integration.shared_storage_configuration.test_managed_filesystem_with_failover.TestManagedFilesystemWithFailover.test_create_filesystem_with_failover_oss_chroma_controlled

Traceback:

```
Traceback (most recent call last):
  File "/usr/share/chroma-manager/tests/integration/shared_storage_configuration/test_managed_filesystem_with_failover.py", line 189, in test_create_filesystem_with_failover_oss_chroma_controlled
    volumes_expected_hosts_in_failover_state,
  File "/usr/share/chroma-manager/tests/integration/core/failover_testcase_mixin.py", line 102, in chroma_controlled_failover
    self.wait_for_commands(self.chroma_manager, failover_target_command_ids)
  File "/usr/share/chroma-manager/tests/integration/core/api_testcase_with_test_reset.py", line 252, in wait_for_commands
    self.wait_for_command(chroma_manager, command_id, timeout, verify_successful)
  File "/usr/share/chroma-manager/tests/integration/core/api_testcase_with_test_reset.py", line 239, in wait_for_command
    self.assertFalse(command["errored"] or command["cancelled"], command)
AssertionError: {u'jobs': [u'/api/job/86/'], u'complete': True, u'created_at': u'2019-05-10T02:04:29.245077', u'message': u'Failing testfs-OST0001 over to secondary', u'cancelled': False, u'errored': True, u'resource_uri': u'/api/command/26/', u'id': 26, u'logs': u'Failed to move target testfs-OST0001_cd4339 to node vm7\n'}
```

sosreports:

[sosreport-vm3-2019-05-10-bsqhszd.tar.xz.zip](https://github.com/whamcloud/integrated-manager-for-lustre/files/3166348/sosreport-vm3-2019-05-10-bsqhszd.tar.xz.zip)
[sosreport-vm5-2019-05-10-zoiherz.tar.xz.zip](https://github.com/whamcloud/integrated-manager-for-lustre/files/3166349/sosreport-vm5-2019-05-10-zoiherz.tar.xz.zip)
[sosreport-vm6-2019-05-10-slophei.tar.xz.zip](https://github.com/whamcloud/integrated-manager-for-lustre/files/3166350/sosreport-vm6-2019-05-10-slophei.tar.xz.zip)
[sosreport-vm7-2019-05-10-yxmbeas.tar.xz.zip](https://github.com/whamcloud/integrated-manager-for-lustre/files/3166351/sosreport-vm7-2019-05-10-yxmbeas.tar.xz.zip)
[sosreport-vm8-2019-05-10-wipzxoj.tar.xz.zip](https://github.com/whamcloud/integrated-manager-for-lustre/files/3166352/sosreport-vm8-2019-05-10-wipzxoj.tar.xz.zip)

**Comment by jgrund** [comment]
@utopiabound Can you take a look at this one?

**Comment by utopiabound** [comment]
vm7 messages file:
```
May 10 02:04:29 vm7 Lustre(testfs-OST0001_cd4339)[10211]: INFO: Starting to mount /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_target4
May 10 02:04:29 vm7 kernel: LDISKFS-fs (sdc): Unrecognized mount option "context="system_u:object_r:tmp_t:s0"" or missing value
May 10 02:04:29 vm7 kernel: LustreError: 10226:0:(osd_handler.c:7616:osd_mount()) testfs-OST0001-osd: can't mount /dev/sdc: -22
May 10 02:04:29 vm7 kernel: LustreError: 10226:0:(obd_config.c:559:class_setup()) setup testfs-OST0001-osd failed (-22)
May 10 02:04:29 vm7 kernel: LustreError: 10226:0:(obd_mount.c:202:lustre_start_simple()) testfs-OST0001-osd setup error -22
May 10 02:04:29 vm7 kernel: LustreError: 10226:0:(obd_mount_server.c:1905:server_fill_super()) Unable to start osd on /dev/sdc: -22
May 10 02:04:29 vm7 kernel: LustreError: 10226:0:(obd_mount.c:1608:lustre_fill_super()) Unable to mount  (-22)
```
This appears to be a duplicate of #925

**Comment by jgrund** [comment]
Interesting, anything else we can derive from this one? It's currently live on trevis-235.

**Comment by jgrund** [comment]
FWIW, checking the lustre data:

```
checking for existing Lustre data: found
Reading CONFIGS/mountdata

   Read previous values:
Target:     testfs-OST0001
Index:      1
Lustre FS:  testfs
Mount type: ldiskfs
Flags:      0x2
              (OST )
Persistent mount opts: ,errors=remount-ro,context="system_u:object_r:tmp_t:s0"
Parameters:  mgsnode=10.73.20.21@tcp:10.73.20.22@tcp failover.node=10.73.20.23@tcp
```

**Comment by jgrund** [comment]
Fixed by #935.
